### ISSUE TITLE
Better document how-to-contribute part

### ISSUE THREAD

**Issue by ekatef** [pypsa-meets-earth/pypsa-earth-lit#12]
It would be very handy to have:
- [ ] explanation on how to make PR, [the hand-on notebook](https://github.com/pypsa-meets-earth/documentation/blob/main/hackathon/0_github_exercise_hands_on.ipynb) from pypsa-earth hackaton can be used for this;
- [ ] add some ready-to-use git spells for the most frequent use cases (like definition of remotes, making a PR upon an existing PR)

**Comment by ekatef** [comment]
As discussed with @tacwebservices and @asolavi, it could make sense to add a brief explanation on the ideal contribution workflow, in particular on using branches to avoid merge conflicts when collaborating

**Comment by asolavi** [comment]
> As discussed with @tacwebservices and @asolavi, it could make sense to add a brief explanation on the ideal contribution workflow, in particular on using branches to avoid merge conflicts when collaborating

Good point, @ekatef. @tacwebservices and I can share our experience once we've tried it and collaborate in the points you are making.

**Comment by ekatef** [comment]
> > As discussed with @tacwebservices and @asolavi, it could make sense to add a brief explanation on the ideal contribution workflow, in particular on using branches to avoid merge conflicts when collaborating
> 
> Good point, @ekatef. @tacwebservices and I can share our experience once we've tried it and collaborate in the points you are making.

@asolavi it would be very helpful! Looking forward for comments from you and @tacwebservices :)
